components/overledger/overledger.app.mjs

```diff
@@ -58,6 +97,7 @@ export default {
     }) {
       return this._makeRequest({
         method: "DELETE",
+        baseUrl: this._getBaseUrl(),
```

Comment:
This repetition is unnecessary, the method already receives the environment in the opts param

Reply:
done